docs/index.txt

```diff
@@ -181,29 +154,63 @@ Compatibility
 JDBC
 ----
 
-This JDBC driver complies to the `JDBC 4.1`_ standard as good as possible and reasonable for
-a Crate driver. The following mandatory features are not supported:
+This JDBC driver complies to the `JDBC 4.1`_ standard as good as possible and
```

Comment:
should be as well as possible but don't like the whole phrase...
Maybe: This JDBC driver follows the JDBC 4.1 standard as it uses PGSQL driver as the underlying connector
but certain features are not fully compatible with Crate